spec/index.html

```diff
@@ -521,13 +521,14 @@ <h3>Quoted Literals</h3>
         show:218 rdfs:label "That Seventies Show"^^<http://www.w3.org/2001/XMLSchema#string> . # same as above
         show:218 rdfs:label "That Seventies Show" .                                            # same again
         show:218 show:localName "That Seventies Show"@en .                 # literal with a language tag
-        show:218 show:localName "That Seventies Show"@en--ltr .            # literal with a language tag and base direction
+        show:218 show:localName "HTML היא שפת סימון."@he--rtl .            # literal with a language tag and base direction
         show:218 show:localName "That Seventies Show"@en-us--ltr .         # literal with a language tag, region subtag, and base direction
         show:218 show:localName 'Cette Série des Années Soixante-dix'@fr . # literal delimited by single quote
         show:218 show:localName "Cette Série des Années Septante"@fr-be .  # literal with a region subtag
         show:218 show:blurb '''This is a multi-line                        # literal with embedded new lines and quotes
```

Comment:
Moving the comment *outside* the literal.
```suggestion
        show:218 show:blurb '''This is a multi-line
```